aspnetcore/tutorials/build-your-first-razor-components-app/samples/3.x/Blazor/Shared/SurveyPrompt.cshtml

```diff
@@ -1,15 +1,15 @@
 ﻿<div class="alert alert-secondary mt-4" role="alert">
-    <span class="oi oi-pencil mr-2" aria-hidden="true"></span> 
+    <span class="oi oi-pencil mr-2" aria-hidden="true"></span>
     <strong>@Title</strong>
 
     <span class="text-nowrap">
         Please take our
-        <a target="_blank" class="font-weight-bold" href="https://go.microsoft.com/fwlink/?linkid=2041371">brief survey</a>
+        <a target="_blank" class="font-weight-bold" href="https://go.microsoft.com/fwlink/?linkid=2077509">brief survey</a>
```

Comment:
This survey could be a lot more useful if it included an opportunity to provide text feedback